### DISCUSSION TITLE
`github.com/golang/protobuf` is deprecated

### DISCUSSION THREAD

**Discussion by kvnzhang** [argoproj/argo-workflows#13438] [Q&A] [answered]
Module `github.com/golang/protobuf v1.5.4` is still in use in argo-workflows main branch, but according to https://pkg.go.dev/github.com/golang/protobuf web page, this module is deprecated and it is suggested to use the `google.golang.org/protobuf` module instead. So is there a plan to do this kind replacement as suggested?

**Comment by agilgur5** [comment, marked as answer]
Yes, there's a closely related issue to this: #11053.

We may also end up removing gRPC entirely which would make protobufs no longer needed either (they're only used for the Server)
